conceptnet5/builders/retrofit_glove.py

```diff
@@ -11,74 +12,39 @@
 from conceptnet5.util import get_data_filename
 # FIXME: negate_concept should live somewhere nicer
 from conceptnet5.builders.assoc_to_vector_space import negate_concept
+from conceptnet5.builders.glove import load_glove_vectors
 
 from collections import defaultdict
 import pathlib
 
-
-def conceptnet_normalizer(text):
+class SparseMatrixBuilder:
     """
-    Normalizes a text into a concept URI. This function assume the text is
-    english.
+    SparseMatrixBuilder is a utility class that helps build a matrix of
+    unknown shape.
     """
```

Comment:
Is it possible that this could reuse the code in assoc_space.SparseEntryStorage?